tools/generators/files_and_groups/src/ElementCreator/CreateGroupChild.swift

```diff
@@ -189,11 +189,14 @@ extension GroupChild.ElementAndChildren {
     init(
         bazelPath: BazelPath,
         element: Element,
+        includeInBazelPathAndIdentifiers: Bool = true,
```

Comment:
Might be nice to mention that the children are included but this controls the element's identifier?

Reply:
Ahh, yeah, good call.